README.md

```diff
@@ -1,2 +1,12 @@
 # Obsidian-DnD-5e-Spells
 Trying to fetch DnD 5e spells using API, then formating the spells to md files for Obsidian in a certain location.
+
```

Comment:
usually in the readme we add the command line arguments (can be the output of --help)